merino/jobs/navigational_suggestions/domain_metadata_uploader.py

```diff
@@ -16,7 +16,7 @@ class DomainMetadataUploader:
     """Upload the domain metadata to GCS"""
 
     DESTINATION_FAVICONS_ROOT: str = "favicons"
-    DESTINATION_TOP_PICK_FILE_NAME_SUFFIX: str = "top_picks_latest.json"
+    DESTINATION_TOP_PICK_FILE_NAME: str = "top_picks.json"
```

Comment:
Let's make it self-explanatory. 
```suggestion
    DESTINATION_TOP_PICK_FILE_NAME: str = "top_picks_latest.json"
```

Reply:
Ok! I'll change it in the provider too to grab `"top_picks_latest.json"`

Reply:
Ah, though we don't want every single timestamped file to have `_latest`, so I'll create a const for `_latest` and add it only to the latest file, since updating `DESTINATION_TOP_PICK_FILE_NAME` will mean all files have `_latest`.